README.md

```diff
@@ -95,8 +98,27 @@ kind get kubeconfig > ./kube.config
 
 ### How does it work
 
-To better depict what `ats` does and how it executes a test scenario, let's have a look at what happens
-when you execute:
+Each run consist of two stages: bootstrapping and test execution.
+
+#### Bootstrapping
+
+`app-test-suite` automates preparation of a cluster used for testing in the following way:
+
+- if you configured your run with `*-tests-cluster-type kind`, a cluster is created with `kind` tool
```

Comment:
This would be a good place to explain that otherwise `external` can/must be used to rely on an existing cluster. If I understood that right.